gcp/basic-gke-cluster/README.md

```diff
@@ -47,51 +47,51 @@ To run the YAML that this blueprint generates, you need:
 
 ## Information required
 
-This blueprint requires:
+This blueprint requires a:
 
-* GCP project ID (see section below for instructions)
-* A GCP region
+* GCP project ID
+* GCP region
 
-### Creating Google Cloud project
+### Create a Google Cloud project
 
 #### Authenticate to gcloud
-Before configuring gcloud CLI you can check available Zones and Regions nearest to your location
+Before configuring gcloud CLI you can check for available Zones and Regions nearest to your location:
 
 ```sh
 gcloud compute regions list
 
 gcloud compute zones list
 ```
 
-Follow gcloud init and select default Zone Ex. europe-west1, make sure you use the same zone as the one selected during blueprint execution.
+Follow `gcloud init` and select default Zone (for example, `europe-west1`). Make sure you use the same zone as the one selected during blueprint execution.
 
 ```sh
 gcloud init
 ```
 
-> NOTE: You need to have proper permissions and privileges in the GCP account to execute these commands. If you are using a personal account you should be having these as you will be the admin. If you are using a company/enterprise account please check with your account administrator. 
+> Note: You must have the appropriate permissions in the GCP account to execute these commands. If you are using a personal account, you should have these permissions, as you are the admin. If you are using a company/enterprise account, check with your account administrator.
 
 #### Set up environment
 
-> NOTE: If you are creating the Project via GUI instead of below commands, there will be a project number, a project name and a project ID when you initialize the project, and only ID should be exported as `TF_ADMIN` variable.
+> Note: If you are creating the Project using the GUI instead of the commands below, there will be a project number, a project name and a project ID when you initialize the project, and only ID should be exported as `TF_ADMIN` variable.
 
 ```sh
 export TF_ADMIN=[GCP project ID]
 ```
 
 #### Create the GCP Project
 
-Create a new project and link it to your billing account (You could do it from the GCP console GUI as well, in that case skip the below command)
+Create a new project and link it to your billing account. You can alternatively choose to do this from the GCP console GUI. In this case, skip the following command.
```

Comment:
I found this part confusing. I would combine the last two sentences so that it's really clear that the final part only applies to the middle part. Basically, it should be clear that you need to do one thing or the other, not both.

Reply:
I made it better (went a different direction, but think it works nicely.